ce/tests/test_api.py

```diff
@@ -176,6 +199,7 @@ def test_stats_bad_params(populateddb, unique_id, var):
     assert math.isnan(rv[unique_id]['max'])
     assert 'time' not in rv[unique_id]
     assert 'units' not in rv[unique_id]
+    assert 'modtime' not in rv[unique_id]
```

Comment:
Assert types (datetime) as well?

Reply:
This is a ``not in`` check, so there's no type to assert.